### PR TITLE
added: quotes in mainpage

### PR DESCRIPTION
**Pull request by anjali1102**
image

### CODE REVIEW

**Comment by anjali1102** [comment]
image

File: src/pages/MainPage/MainPage.jsx [hunk 1]
```diff
@@ -1,21 +1,52 @@
...
 
   let wallpaper = "https://source.unsplash.com/1920x1080/?wallpaper";
 
+  useEffect(() => {
+    (async () => {
+      try {
+        let res = await axios.get(
+          "https://api.quotable.io/random?maxLength=90"
+        );
+        console.log(res);
```
Comment (dev-enforced) [inline src/pages/MainPage/MainPage.jsx]: Can remove this debugger

File: src/pages/MainPage/MainPage.jsx [hunk 2]
```diff
@@ -1,21 +1,52 @@
-import React from "react";
-import "../../App.css";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import "../MainPage/MainPage.css";
 import { useDate } from "../../hooks/useDate";
 
 const MainPage = () => {
+  const [quote, setQuote] = useState({ content: "", author: "" });
```
Comment (dev-enforced) [inline src/pages/MainPage/MainPage.jsx]: Can store the parameters inside useState as a separate variable named as defaultQuoteData.

File: src/pages/MainPage/MainPage.jsx (hunk 1 above)
Comment (znjs) [inline src/pages/MainPage/MainPage.jsx]: can remove the console

File: src/pages/MainPage/MainPage.jsx [hunk 3]
```diff
@@ -1,21 +1,52 @@
...
+      try {
+        let res = await axios.get(
+          "https://api.quotable.io/random?maxLength=90"
+        );
+        console.log(res);
+        setQuote({
+          content: res.data.content.slice(0, -1),
+          author: res.data.author,
+        });
+        console.log(quote);
```
Comment (znjs) [inline src/pages/MainPage/MainPage.jsx]: can remove the console

**Review by znjs — APPROVED** [review]
Clean UI
Good conventions used

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks clean ,UI also looks clean .you can go ahead  and merge

**Review by dev-enforced — APPROVED** [review]
The live preview is working as expected . The quotes are getting changed on hard refresh. Good work @anjali1102 !!

**Pull request merged**